lib/config.js

```diff
@@ -63,6 +56,9 @@ export function configure(otpConfig) {
           </a>
         </div>
       </div>
-    )
+    ),
+    // See config-example.js for more customization approaches.
```

Comment:
This is a good enough example.

Reply:
Perhaps we can remove `config-example.js` and just point people to otp-react-redux/custom-icons.js`. Thoughts, @evansiroky?

Reply:
A new class can be inlined here. I suppose we can try to document that with an extended example, but it seems like a motivated developer would be able to figure that out on their own.

Reply:
Are we talking about inlining icons inside `config.js`, as in here?
https://github.com/opentripplanner/otp-react-redux/blob/381c3f526368bbfdc1666355dfb1cca8eeae35c9/lib/components/icons/mode-icon.js#L32